app/views/profiles/dashboard/_journey.html.erb

```diff
@@ -19,6 +19,13 @@
               <li> <i class="fa fa-money" aria-hidden="true"></i>  <%= t('.price') %>:€ <%= journey.car.price_hour.to_i %>/hr</li>
               <!-- <li> <i class="fa fa-clock-o" aria-hidden="true"></i>  <%#= t('.duration') %>:<%= journey.duration %> h </li> -->
               <li> <i class="fa fa-users" aria-hidden="true"></i  >  <%= t('.children') %>: <%= journey.remaining_seats %> </li>
+              
+              <% if journey.logs[0] %>
```

Comment:
I could pass an instance variable to the view to make this condition `true/false`, but I think that I would need still to add an `if` statement in the view. Are there any other ways of doing this?